spring-cloud-dataflow-module-deployers/spring-cloud-dataflow-module-deployer-kubernetes/pom.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<parent>
+		<groupId>org.springframework.cloud</groupId>
+		<artifactId>spring-cloud-dataflow-module-deployers-parent</artifactId>
+		<version>1.0.0.BUILD-SNAPSHOT</version>
+	</parent>
+	<modelVersion>4.0.0</modelVersion>
+	<artifactId>spring-cloud-dataflow-module-deployer-kubernetes</artifactId>
+	<name>spring-cloud-dataflow-module-deployer-kubernetes</name>
+	<description>Kubernetes module deployer SPI implementation</description>
+	<packaging>jar</packaging>
+
+	<properties>
+		<kubernetes.client.version>1.3.49</kubernetes.client.version>
+		<kubernetes.client-mock.version>1.3.52</kubernetes.client-mock.version>
+	</properties>
+
+	<dependencies>
+		<dependency>
+			<groupId>org.springframework.cloud</groupId>
+			<artifactId>spring-cloud-dataflow-module-deployer-spi</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.cloud</groupId>
+			<artifactId>spring-cloud-stream-module-launcher</artifactId>
```

Comment:
yes. I haven't actually compiled your code yet, just browsing. But if you need it because you use some API from `spring-cloud-stream-module-launcher`, then this is "wrong" as we want those two to be decoupled.